dashboard/test/models/pd/workshop_test.rb

```diff
@@ -454,8 +454,8 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     workshop.reload.destroy!
 
     assert workshop.reload.deleted?
-    refute Pd::Workshop.exists? workshop.attributes
-    assert Pd::Workshop.with_deleted.exists? workshop.attributes
+    refute Pd::Workshop.exists? id: workshop.id
```

Comment:
We're not 100% sure why this error appeared, but simplifying the test succeeded. Perhaps related to the newly added `properties` column for `pd_workshops`, but unclear why this wouldn't have appeared earlier if that is the culprit.